src/components/navigation/navigation.jsx

```diff
@@ -0,0 +1,27 @@
+import Inferno from 'inferno';
+import Component from 'inferno-component';
+import { Link } from 'inferno-router';
+
+class Navigation extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loggedIn: false,
+    };
+  }
+  render() {
+    return (
+      <div className="bar">
+        <Link className="createCrawl" to="/create">Create a Crawl</Link>
+        <Link className="header" to="/">PubCrawler</Link>
+        <div className="auth">
+          {this.state.loggedIn ? <Link className="element" to="/logout">Logout</Link> :
+          <Link className="element" to="/login">Log in</Link>}
```

Comment:
You can indent as well with eslint not freaking out.

In general I prefer 'single' code lines across multiple lines to be indented for readability. You're call.